react/week2/new-todo/src/Todolist.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import Todoitem from "./Todoitem";
+
+function Todolist({ todos }) {
+  const [itemList, setItemList] = useState(todos);
+
+  function addNewItem() {
+    const newItem = {
+      id: Math.floor(Math.random() * 100),
+      description: "add task",
+    };
+    setItemList(itemList.concat(newItem));
+  }
+    const deleteItem = (id) => {
+      const todos = [...itemList];
+      setItemList(todos.filter((todo) => todo.id !== id));
+      };
```

Comment:
Why did you use const todos = [...itemList];  ??
1. todos variable is already used so you have to use another variable .
2. you can pass this function inside the set. It will not work. 

 here is solution :
    const deleteItem = (id) => {
      const todoList = itemList.filter(todo => todo.id !== id);
      setItemList(todoList);
      };